libparsec/crates/testbed/src/template/events.rs

```diff
@@ -1367,6 +1376,21 @@ impl TestbedEventShareRealm {
         if builder.check_consistency {
             utils::assert_organization_bootstrapped(&builder.events);
             utils::assert_realm_exists(&builder.events, realm);
+
+            match role {
+                None | Some(RealmRole::Reader | RealmRole::Contributor) => (),
+                Some(RealmRole::Owner | RealmRole::Manager) => {
+                    match get_user_current_profile(&builder.events, user) {
+                        UserProfile::Admin | UserProfile::Standard => (),
+                        UserProfile::Outsider => {
+                            panic!(
+                                "User {} is Outsider, realm cannot be shared with him as {:?}",
+                                user, role
```

Comment:
```suggestion
                Some(role @ RealmRole::Owner | RealmRole::Manager) => {
                    match get_user_current_profile(&builder.events, user) {
                        UserProfile::Admin | UserProfile::Standard => (),
                        UserProfile::Outsider => {
                            panic!(
                                "User {user} is Outsider, realm cannot be shared with him as {role}"
```

Use a binding to remove the debug print

Reply:
:+1: done.